packages/mocha/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+The contents of this package were previosly published as part of [`@metamask/eslint-config`](https://npmjs.com/package/@metamask/eslint-config).
+For changes prior to version `6.0.0`, please refer to the changelog of that package.
+
+[Unreleased]:https://github.com/MetaMask/eslint-config/compare/v5.0.0...HEAD
+<!-- [6.0.0]:https://github.com/MetaMask/eslint-config/tree/v6.0.0 -->
```

Comment:
What is this comment for :thinking:

Reply:
Just a placeholder for the upcoming v6 release of all these packages. I can just remove it for now if we care 😛